File: python/tannu/gradecard.py
marks=int(input("enter your marks"))
if marks<=90:
    if marks<=80:
        if marks<=70:
            if marks<=60:
                if marks<=50:
                    if marks<=40:
                        if marks<=33:
                            print("you have failed")
                        else:
                            print("your grade is D")
                    else:
                        print("your grade is C2")
                else:
                    print("your grade is C1")
            else:
                print("your grade is B2")
        else:
            print("your grade is B1")
    else :
        print("your grade is A2")
else:
    print("your grade is A1")

File: C&C++/BCA2/Nishita/gradecard.c
#include<stdio.h>
int main()
{
    int arr[6],i,st=0,per;
    printf("Enter the elements of array :- ");
    for(i=0;i<6;i++)
    {
        scanf("%d",&arr[i]);
        st=st+arr[i];
    }

     per=st/6;
     printf("%d is your perchange\n",per);

        if(per>= 80 && per<100)
        {
            printf("You got A grade");
        }
        else if(per>=60 && per<=79)
        {
            printf("you got B grade");
        }    
        else if(per>40 && per<=59)
        {
            printf("you got C grade");
        }
        else if(per>33 && per<=39)
        {
            printf("you got D grade");    
        }
        else
        {
            printf("you got fail");
        }
}
